as/http.js

```diff
@@ -182,6 +183,16 @@ TChannelHTTP.prototype.sendRequest = function send(treq, hreq, options, callback
 
     treq.headers.as = 'http';
     if (treq.streamed) {
+        self.channel.emitFastStat(self.channel.buildStat(
```

Comment:
What I meant to say is that you want to include `sendStream()` in `now() - start` so please add stats after you do IO thingies